src/engines/html5/html5.js

```diff
@@ -149,6 +149,20 @@ export default class Html5 extends FakeEventTarget implements IEngine {
     Html5._el.load();
   }
 
+  /**
+   * The player playback rates.
+   * @type {Array<number>}
+   */
+   PLAYBACK_RATES = [0.5, 1, 2, 4];
+
+   /**
+   * The player playback rates.
+   * @type {Array<number>}
+   */
+   LIVE_PLAYBACK_RATES = [1];
```

Comment:
should be `static const`

Reply:
also fix the jsdoc. also above

Reply:
@itaykinnrot why differentiate live and vod?

Reply:
@OrenMe  to keep the same behaviour we have today - i removed it from the ui

Reply:
The reason it was in the UI is that we don’t want to limit the API of player with BL.
The UI in this case is opinionated and can make assumptions but from playback perspective having rate higher or lower than 1 for live is also possible(DVR playback).
Please keep this logic in UI.